### ISSUE TITLE
Dependencies are not properly loaded

### ISSUE THREAD

**Issue by mosteo** [alire-project/alire#79]
Issuing `alr show hello` shows no dependencies:
```
hello=1.0.1: "Hello, world!" demonstration project
Origin: commit 65725c20778875eef12b61a01b437120932965f3 from https://github.com/alire-project/hello.git
Available when: True
Properties:
   Maintainer: alejandro@mosteo.com
```

whereas the pre-toml version showed:
```
hello=1.0.1: "Hello, world!" demonstration project
Origin: commit 65725c20778875eef12b61a01b437120932965f3 from https://github.com/alire-project/hello.git
Dependencies (direct):
   libhello is Within_Major (1.0.0)
```

**Comment by pmderodat** [comment]
Running `alr show hello` with the current master branch, I don’t even get your output:
```
alr show hello
Not found: hello with Newest version
```
Debugging quickly reveals that it’s because the `show` command does not try to load the index. I had the same issue with the `search` command before my previous pull request. I guess for this specific case we don’t need to load the full index (just `hello`), but that’s an optimization, so to be kept for later. :-) I’ll send a patch for this as well.

**Comment by mosteo** [comment]
Yes, it's my fault that the fix for show loading the index didn't make to master, so you ended fixing it again. Sorry about that.

I will merge #82 next and try to separate #81 into meaningful PRs, so at least you can take a look at commits that involve core functionality without much distraction.

**Comment by mosteo** [comment]
Fixed by #82, closing
